Testscripts/Linux/SR-IOV-Utils.sh

```diff
@@ -76,7 +76,13 @@ VerifyVF()
         vf_interface=$(ls /sys/class/net/ | grep -v 'eth0\|eth1\|lo' | head -1)
     else
         synthetic_interface=$(ip addr | grep "$VF_IP1" | awk '{print $NF}')
-        vf_interface=$(find /sys/devices/* -name "*${synthetic_interface}" | grep "pci" | sed 's/\// /g' | awk '{print $12}')
+        if [[ $DISTRO_VERSION =~ 6\. ]]; then
+            synthetic_MAC=$(ip link show ${synthetic_interface} | grep ether | awk '{print $2}')
```

Comment:
I think this can simpler than those lines. can you also use regex or something similar to the list 84?

Reply:
Changed to one line command

Reply:
I meant regex or something better approach not concise the commands in single line.

Reply:
Taking reference from existing code in script - 
vf_interface=$(find /sys/devices/* -name "*${synthetic_interface}" | grep "pci" | sed 's/\// /g' | awk '{print $12}')

I have added the way which is widely used in the LISAv2 with grep and awk and there is no regex in line 84. sed is used to replace / with space

Reply:
This is widely used in LISAv2? ok, let me check out later.